README.md

```diff
@@ -83,15 +83,20 @@ EOF
 $ sudo apt install node watchman openjdk-17-jdk
 ```
 
-## Android
+# Running the app locally
+
+Do **not** use the `a` or `i` subcommands in `npm run start`, they don't work with our workflow.
```

Comment:
Ohh I thought it was specifically the `s` subcommand that does cloud stuff and doesn't work with our workflow, was that incorrect?

Reply:
`s` is kind of a no-op, it switches between expo go and dev builds, neither of which we use, so `a` or `i` don't work reliably in either mode